### PR TITLE
Handle error during connection in sleep mode.

### PR DESCRIPTION
**Pull request by ngraziano**
If TV is in sleep mode (not totally off) when I try to connect, its response  is :
```json
{"error":[7,"not power-on"],"id":1}
```
and there is not the necessary cookie for authentication but `is_connected()` return `True` 

I get sleep mode for few minutes when I turn the TV off.

### CODE REVIEW

File: braviarc/braviarc.py [hunk 1]
```diff
@@ -71,8 +71,10 @@ def connect(self, pin, clientid, nickname):
 
         else:
             _LOGGER.info(json.dumps(response.json(), indent=4))
-            self._cookies = response.cookies
-            return True
+            resp = json.loads(response.content.decode('utf-8'))
```
Comment (antonioparraga) [inline braviarc/braviarc.py]: does response always have a content attribute snd content is alwais not null at this point?

**Review by antonioparraga — COMMENTED** [review]
good fix, however please see my question

**Comment by ngraziano** [comment]
I change to use response.json(), it was use before for log so this should be good.
I also change the test to be allow null response.

File: braviarc/braviarc.py [hunk 2]
```diff
@@ -70,9 +70,11 @@ def connect(self, pin, clientid, nickname):
             return False
 
         else:
-            _LOGGER.info(json.dumps(response.json(), indent=4))
-            self._cookies = response.cookies
-            return True
+            resp = response.json()
+            _LOGGER.info(json.dumps(resp, indent=4))
```
Comment (antonioparraga) [inline braviarc/braviarc.py]: Should be debug instead of info at this point?

**Review by antonioparraga — CHANGES_REQUESTED** [review]

**Comment by antonioparraga** [comment]
perfect!
